packages/bundle/src/boot/exports/internal.ts

```diff
@@ -0,0 +1,11 @@
+// We should review exports in this file to make sure 1P = 3P.
```

Comment:
We're lucky there is no any context involved as otherwise we'd have multiple context instances between `bundle` and `bundle/internal`

Reply:
Oh yea... it's lucky that we don't have to export `<*Context>` from `api`, and re-exporting it here. That's a good thinking and future catch.